src/ncp/ncp_base_dispatcher.cpp

```diff
@@ -948,6 +980,9 @@ NcpBase::PropertyHandler NcpBase::FindInsertPropertyHandler(spinel_prop_key_t aK
 
 #if OPENTHREAD_FTD
 #if OPENTHREAD_ENABLE_COMMISSIONER
+    case SPINEL_PROP_MESHCOP_COMMISSIONER_JOINERS:
```

Comment:
Noticing that some of commissioner related code are wrapped with `OPENTHREAD_ENABLE_COMMISSIONER` and some are wrapped with `OPENTHREAD_FTD` as well. We might want to unify it.
MTD should also be commissioner-capable, right? @jwhui

Reply:
Good point. Currently the commissioner APIs and implementation are tied to `FTD`.
- In the public APIs: [commissioner_api.cpp](https://github.com/openthread/openthread/blob/master/src/core/api/commissioner_api.cpp#L46)
- And in core implementation: [commissioner.cpp](https://github.com/openthread/openthread/blob/master/src/core/meshcop/commissioner.cpp#L57)

To be consistent, will use `#if OPENTHREAD_FTD && OPENTHREAD_ENABLE_COMMISSIONER` in NCP code for any commissioner related logic (where don't already have `FTD` check). Thanks

Reply:
I am fine to keep the tie in this PR. We may raise a separate issue to discuss whether to remove such tie. Thanks.